### DISCUSSION TITLE
How to Setting only Cmine

### DISCUSSION THREAD

**Discussion by linhvo412** [hldh214/lok_bot#39] [Q&A] [answered]
I just Cmine. How to setting bot only for Cmine? Please help me, thank you so much!!!!

**Comment by EichmannT** [comment, marked as answer]
Adjust the config.json file to set what you want on and off with TRUE and FALSE.
This piece of the config does it.
{
        "name": "socf_thread",
		"comment":"Farm crystals and goblins only in kw arguments list here. Need to build a farming bot with the object_code_list",
        "enabled": true,
        "kwargs": {
          "object_code_list": [
            20100105,
            20200104
          ],
          "radius": 8
        },
        "interval": {
          "start": 1,
          "end": 1
        }
      }

That's my comment code there, not the original. I've read all the code and commented in English to myself and started building a TODO list.
Those object codes are the Goblin and Crystal Mine. Delete 20200104 from the list (Goblin) and leave only 20100105 (CMine) there if you just want to farm crystals. You'll need to adjust the radius if you aren't in a spawn area. Code runs every minute by default according to another thread. Crystal mines spawn every 60 min. Just be careful with how often you scan for mines or the account will get banned.

**Reply by linhvo412** [reply]
thank you so much!!!
